ddtrace/_monkey.py

```diff
@@ -157,10 +157,12 @@ def on_import(hook):
         except Exception:
             if raise_errors:
                 raise
-            log.error("failed to import ddtrace module %r when patching on import", path, exc_info=True)
+            error_msg = "failed to import ddtrace module %r when patching on import" % (path,)
+            log.error(error_msg, exc_info=True)
+            telemetry_lifecycle_writer.add_integration(module, False, PATCH_MODULES.get(module) is True, error_msg)
```

Comment:
Previously we only queued integrations that were successfully patched. Now we also collect data about integrations that failed to patch